Added SROCC, SRCCL and AR5 WGII (parts A and B)

pmarbaix opened the pull request:
Added SROCC, SRCCL and AR5 WGII (full report, parts A global and B regional). Thanks for all the work already done!

pmarbaix committed:
Added draft SROCC bibliography

I did not find the SROCC bibliography in BibTex anywhere, so I did this draft conversion based on https://www.ipcc.ch/srocc/cite-report/

pmarbaix committed:
Added SROCC

pmarbaix committed:
Updated README.md to include the SROCC

pmarbaix committed:
Added SRCCL

pmarbaix committed:
Fixed the Makefile and srccl.bib errors

pmarbaix committed:
Updated README.md to include SRCCL

pmarbaix committed:
Create ar5-wg-ii.bib

Ref for full AR5 WGII rep, as found in https://www.ipcc.ch/site/assets/uploads/2018/02/WGIIAR5-FrontMatterA_FINAL.pdf

pmarbaix committed:
Merge branch 'main' into pmarbaix-srocc

pmarbaix committed:
Merge pull request #1 from pmarbaix/pmarbaix-srocc

Added draft SROCC bibliography

pmarbaix committed:
fixed typo

pmarbaix committed:
Update README.md

pmarbaix committed:
Update ar5-wg-ii.bib: corrected mistakes & added part B

rgieseke commented:
Thank you for contributing, will take a look later!

rgieseke committed:
Fix makefile and add links

rgieseke committed:
Fix Makefile

rgieseke committed:
Update docs

rgieseke commented:
I pushed some changes to the doc creation process which should make it easier to review the changes. There were some warnings about fields from the BibTeX processing.

pmarbaix commented:
Thanks for the improvements. Regarding the warning, do you mean things like (?):
```
Database file #1: ../srocc.bib
Warning--can't use both author and editor fields in IPCC_2019_SROCC
```
If so, I have the impression that these are already present in your repository, and I followed the same approach. For example, if I create your files with `make -B`(= Unconditionally make all targets), I get
```
Database file #1: ../ar6-wg-i.bib
Warning--can't use both author and editor fields in IPCC_2021_WGI
```
This is because the reference 'IPCC_2021_WGI' is of type @book, while this type does not allow the authors and editors field simultaneously. I do not know why it is so, and could not find a clear documentation about it. https://ctan.mc1.root.project-creative.net/info/bibtex/tamethebeast/ttb_en.pdf indicates that @book requires an author OR an editor, but does not say that both cannot be provided simultaneously. As far as I understand, the warning does not appear with @incollection. 

I am not familiar with GitHub yet, so I apologise if I missed something, but this is all I understand so far.
If I understand what you mean / want and the problem itself, then we have two choices: either continuing to live with that warning (it is nothing severe and some software do use both authors and editors), or we have to change @book to something else for all reports. I hope that this helps.

pmarbaix committed:
Added TAR wgii SPM

pmarbaix committed:
Re-ordered chronologically + added reports in README.md

pmarbaix committed:
Added TAR wgii SPM in Makefile

pmarbaix committed:
Merge branch 'main' of https://github.com/pmarbaix/ipcc-bibtex

rgieseke commented:
Yeah, the warnings could be something to fix in general or maybe it's not possible. I spotted some in the new entries but haven't looked into it in detail.

One thing I saw while skimming was

 	R. (ed.) van Diemen. Annex i: Glossary. In P.R. Shukla, J. Skea, E. Calvo, V. Masson-Delmotte, H.-O. Pörtner, D. C., P. Zhai, R. Slade, S. Connors, R. van, M. Ferrat, E. Haughey, S. Luz, S. Neogi, M. Pathak, J. Petzold, J. Portugal, P. Vyas, E. Huntley, K. Kissick, M. Belkacemi, and J. and Malley, editors, Climate Change and Land: an IPCC special report on climate change, desertification, land degradation, sustainable land management, food security, and greenhouse gas fluxes in terrestrial ecosystems, book section Annex I Glossary. Cambridge University Press, Cambridge, UK and New York, NY, USA, 2019. 
 	
 in SRCCL. I think this should have IPCC as the author as van Diemen as the coordinating editor.

pmarbaix committed:
Fixed author / editors for the glossary

pmarbaix commented:
> in SRCCL. I think this should have IPCC as the author as van Diemen as the coordinating editor.

Right, I corrected this. It is sometimes hard to get the official bib reference for IPCC reports in BibTeX. I hope that it is now good enough for the pull, thanks again.
Philippe.

rgieseke commented:
Thank you @pmarbaix! With the change in the `sed` command (see review comment) I can't build it locally on Linux. Can you confirm you are on Mac? I can also look into a solution that might work for both platforms (maybe with an environment variable or a check in the Makefile).

pmarbaix commented:
I apologise for the change to the Makefile (sed). I did not want to push it on GitHub but apparently made a mistake. Indeed it is needed on Mac. Of course a solution that would check and work on both would be wonderful (if that is not too long as compared to the objective - getting refs for the IPCC reports). The simple solution would be to remove my changes on the Makefile, as these were not intentional.

pmarbaix committed:
Removed the Mac-specific argument in sed (reverted to the standard version)

rgieseke commented:
Thank you! This builds now for me again locally. I'll review this and will then merge.

rgieseke commented:
Sorry for not having yet reviewed this.
I guess this being somewhat large has led me to not knowing how long it will take to not starting ... :slightly_smiling_face: 

I'll split it up:

- [x] Review SROCC
- [ ] Review SRCCL
- [ ] Review AR5 WGII (parts A and B)
- [ ] Review TAR WGII SPM
- [ ] Fix merge conflicts

pmarbaix commented:
Thanks a lot, but if I can tell politely about my humble feelings: it seems to require a lot of time. This morning I go another reference to an IPCC document (2023 scenario workshop) perfectly converted to BibTeX by an AI chatbot… I don’t want to dismiss your efforts and help, thanks again, but how we can be efficient is not clear to me. Crossing fingers that this reach an happy and not too complicated result!

Best,

Philippe.

Le 5 avr. 2024 à 11:49, Robert Gieseke ***@***.***> a écrit :



Sorry for not having yet reviewed this.
I guess this being somewhat large has led me to not knowing how long it will take to not starting ... 🙂

I'll split it up:

  *    Review SROCC
  *    Review SRCCL
  *    Review AR5 WGII (parts A and B)
  *    Review TAR WGII SPM
  *    Fix merge conflicts

—
Reply to this email directly, view it on GitHub<https://github.com/openclimatedata/ipcc-bibtex/pull/35#issuecomment-2039365928>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABF7ATQP7GSZYDMKHNGZPILY3ZXS3AVCNFSM6AAAAABFDAO3R2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAMZZGM3DKOJSHA>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

rgieseke commented:
I haven't had much downtime or long train rides to properly review this, so if anyone likes to help with that ...

pmarbaix committed:
Update ar6-wg-ii.bib

Removed the entry for AR6 WGII Chapter 7 Supplementary Material: it does not exist.
